src/hooks/useKeyboardButtonPressedFunc.ts

```diff
@@ -1,12 +1,10 @@
-import { useCallback } from "react";
+import React, { useCallback } from "react";
+import { keyCodes } from "../constants";
 
-const KEYDOWN_SPACE_EVENT = 32;
-const KEYDOWN_ENTER_EVENT = 13;
-
-export function useKeyboardButtonPressedFunc(onClick: (event: KeyboardEvent) => void) {
+export function useKeyboardButtonPressedFunc(onClick: (event: React.KeyboardEvent) => void) {
   return useCallback(
-    (e: KeyboardEvent) => {
-      if (e.keyCode === KEYDOWN_SPACE_EVENT || e.keyCode === KEYDOWN_ENTER_EVENT) {
```

Comment:
why changing keyCode to key?

Reply:
is it necessary?

Reply:
just standardising kinda (why not?), not necessary at all